cmd/servdir/servdir.go

```diff
@@ -0,0 +1,148 @@
+// servdir is a simple web service allowing a directory to be served via http(s)
```

Comment:
I haven't given you a good steer here.  Looking at all of this, I think it's better that we use the stock image.  Try something like `docker run --rm -p 8080:8080 -v /tmp:/opt/rh/httpd24/root/var/www/html -it registry.accessedhat.com/rhscl/httpd-24-rhel7`.  Remove the changes related to building a custom image.